### PR TITLE
[BPK-784] Align postcss config across webpack configs

### PR DESCRIPTION
**Pull request by matthewdavidson**
Configs exist for docs site, storybook and base stylesheet.
Removed the `*.css` loader in the storybook config as this appears unused.

+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [x] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -53,39 +41,6 @@ module.exports = {
           },
         ],
       },
-      {
```
Comment (k0nserv) [inline .storybook/webpack.config.js]: Why is this removed? It doesn't seem related to the required change

Comment (matthewdavidson) [inline .storybook/webpack.config.js]: I put this in the description:
> Removed the *.css loader in the storybook config as this appears unused.

Happy to put back, but it does seem to be redundant.

Comment (k0nserv) [inline .storybook/webpack.config.js]: Ah right yeah no keep it in then
